Explorer/Assets/DCL/Chat/ChatEntryView.cs

```diff
@@ -41,10 +45,13 @@ public class ChatEntryView : MonoBehaviour
 
         public void SetUsername(string username, string walletId)
         {
-            playerName.text =
-                string.IsNullOrEmpty(walletId) ?
-                    username : $"{username}#<color=#76717E>{walletId.Substring(0,5)}</color>";
-            verifiedIcon.gameObject.SetActive(string.IsNullOrEmpty(walletId));
+            playerName.text = username.Contains("#")
```

Comment:
You execute `username.Contains("#")` 3 times. You should reuse the result. Moreover `IndexOf` can act as `Contains` if compared with `-1`